### PR TITLE
Decrease log level from warn to info for git clone reject

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/167386

### CODE REVIEW

File: lib/OpenQA/Task/Git/Clone.pm [hunk 1]
```diff
@@ -65,7 +65,7 @@ sub _git_clone ($app, $job, $ctx, $path, $url) {
 
         my $origin_url = $git->get_origin_url;
         if ($url ne $origin_url) {
-            $ctx->warn("Local checkout at $path has origin $origin_url but requesting to clone from $url");
+            $ctx->info("Local checkout at $path has origin $origin_url but requesting to clone from $url");
```
Comment (okurz) [inline lib/OpenQA/Task/Git/Clone.pm:68]: Can you rephrase the message to make the "so what" clear?

**Review by okurz — CHANGES_REQUESTED** [review]

**Comment by perlpunk** [comment]
webui-docker-compose fail is due to https://progress.opensuse.org/issues/167524

File: lib/OpenQA/Task/Git/Clone.pm (hunk 1 above)
Comment (kalikiana) [reply lib/OpenQA/Task/Git/Clone.pm:68]: Is this actionable? My understanding from the ticket is this works as expected. So I would actually say remove it.

Comment (Martchus) [reply lib/OpenQA/Task/Git/Clone.pm]: This can be helpful without being actionable, e.g. when debugging and trying to make sense of what our Git code does or skips.

File: lib/OpenQA/Task/Git/Clone.pm [hunk 2]
```diff
@@ -65,7 +65,9 @@ sub _git_clone ($app, $job, $ctx, $path, $url) {
 
         my $origin_url = $git->get_origin_url;
         if ($url ne $origin_url) {
-            $ctx->warn("Local checkout at $path has origin $origin_url but requesting to clone from $url");
+            $ctx->info(
+"Local checkout at $path has origin $origin_url but requesting to clone from $url. This is not that bad."
```
Comment (Martchus) [inline lib/OpenQA/Task/Git/Clone.pm:69]:
```suggestion
"Local checkout at $path has origin $origin_url but requesting to clone from $url. Be aware that the requested URL will not be cloned."
```

Comment (perlpunk) [inline lib/OpenQA/Task/Git/Clone.pm:69]: Suggestion by @okurz ;-)
```suggestion
"Local checkout at $path has origin $origin_url but requesting to clone from $url. This is fine.🔥"

```

**Review by okurz — APPROVED** [review]
Well, let's try :)

File: lib/OpenQA/Task/Git/Clone.pm (hunk 2 above)
Comment (perlpunk) [reply lib/OpenQA/Task/Git/Clone.pm]: Sorry, that was just meant to be a joke, not a real suggestion

**Review by Martchus — APPROVED** [review]

File: lib/OpenQA/Task/Git/Clone.pm (hunk 2 above)
Comment (asdil12) [reply lib/OpenQA/Task/Git/Clone.pm]: Pssst. Don't prevent the opinion to see what we can break with UTF-8 characters on the logfile ;)
